src/MutatorOrchestrator.ts

```diff
@@ -64,13 +62,21 @@ export default class Mutator {
    * @returns {Mutant[]} All possible Mutants for the given set of nodes.
    */
   private findMutants(sourceFile: string, originalCode: string, ast: ESTree.Program, nodes: any[]) {
-    var mutants: Mutant[] = [];
-    _.forEach(nodes, (astnode, index) => {
+    let mutants: Mutant[] = [];
+    nodes.forEach((astnode) => {
       if (astnode.type) {
-        _.forEach(this.mutations, (mutation: BaseMutation) => {
-          if (mutation.canMutate(astnode)) {
-            mutants = mutants.concat(mutation.applyMutation(sourceFile, originalCode, astnode, ast));
+        Object.freeze(astnode);
+        this.mutators.forEach((mutator: Mutator) => {
+          try {
+            mutator.applyMutations(astnode, _.cloneDeep).forEach((mutatedNode: ESTree.Node) => {
+              let mutatedCode = parserUtils.generate(mutatedNode);
+              let originalNode = nodes[(<StrykerNode>mutatedNode).nodeID];
+              mutants.push(new Mutant(mutator, sourceFile, originalCode, mutatedCode, originalNode.loc));
```

Comment:
Maybe add a debug log line here?
`generated 3 mutants on using source line 23 using the RemoveCondition mutator` for example?